psutil/__init__.py

```diff
@@ -357,6 +363,12 @@ def __init__(self, seconds, pid=None, name=None):
 if POSIX:
     from . import _psposix
     _psposix.TimeoutExpired = TimeoutExpired
+if CYGWIN:
+    from . import _pslinux
+    _pslinux.NoSuchProcess = NoSuchProcess
+    _pslinux.ZombieProcess = ZombieProcess
+    _pslinux.AccessDenied = AccessDenied
+    _pslinux.TimeoutExpired = TimeoutExpired
```

Comment:
I don't like this.  Didn't there used to be a `psutil._exceptions` module?  Why did that go away?  Having something like that would seem a bit less crazy like this...